packages/connector-sendgrid-mail/README.md

```diff
@@ -1,2 +1,86 @@
-### SendGrid Mail README
-placeholder
+# SendGrid mail connector
+
+SendGrid (also known as Twilio SendGrid) is a customer communication platform for transactional and marketing email. We can use its email sending function to send a "verification code" (or in other words, passcode).
```

Comment:
```suggestion
SendGrid (a.k.a. Twilio SendGrid) is a customer communication platform for transactional and marketing email. We can use its email sending function to send a "verification code".
```

Reply:
Should align the description sentence of your Twillos doc.  This is more straightforward.